src/Core/Tools/Repositories/ISendRepository.cs

```diff
@@ -1,10 +1,36 @@
-﻿using Bit.Core.Repositories;
+﻿#nullable enable
+
+using Bit.Core.Repositories;
 using Bit.Core.Tools.Entities;
 
 namespace Bit.Core.Tools.Repositories;
 
+/// <summary>
+/// A repository for loading <see cref="Send"/>s.
```

Comment:
nit: `A repository for interacting with <see cref="Send"/>s.` since `IRepository` does bring methods for creating and updating them.

Reply:
"interacting with sends" is what the whole system does, so I scoped the doc specifically to loads and saves.

Reply:
But since `ISendRepository` inherits from `IRepository` anything that implements it will be able to do things from the whole system, I feel like that's helpful to mention in its comment. I could maybe see an argument for that if we only did an explicit implementation of the interface but we very rarely (never?) do that in this repo.

Reply:
There's all kinds of ways to interact with sends once you have an instance, yes. But the only thing the repository does is load and save. There's special ways to do that (e.g. upsert/replace/create/etc), but at the end of the day the repo's scope is handling persistence of the send data. Anything beyond that, and you shouldn't use the repo to do it.